docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -8,25 +8,48 @@ description: How to configure graphical database clients for Teleport Database A
 This article describes how to configure popular graphical database clients to
 work with Teleport Database Access.
 
-## Obtain local certificate/key-pair
+## Get connection information
 
-Issue the following command after you login with `tsh`:
+<Tabs>
+<TabItem label="With TLS routing">
+If you're using Teleport in [TLS routing](../../setup/operations/tls-routing.mdx)
+mode where each database protocol is multiplexed on the same web proxy port, use
+the following command to start a local TLS proxy your GUI database client will
+be connecting to:
+
+```code
+$ tsh proxy db <database-name>
+Started DB proxy on 127.0.0.1:61740
+
+Use following credentials to connect to the <database-name> proxy:
+  ca_file=/Users/r0mant/.tsh/keys/root.gravitational.io/certs.pem
+  cert_file=/Users/r0mant/.tsh/keys/root.gravitational.io/alice-db/root/<database-name>-x509.pem
+  key_file=/Users/r0mant/.tsh/keys/root.gravitational.io/alice
+```
+
+Use the displayed local proxy host/port and credentials paths when configuring
+your GUI client below. When entering the hostname, use `localhost` rather than
+`127.0.0.1`.
+</TabItem>
+<TabItem label="Without TLS routing">
```

Comment:
nit: "TLS routing" and "Separate port" instead of "With TLS routing" and "Without TLS routing"